### PR TITLE
Set forceDelete to false for deleteTaskQueue

### PR DESCRIPTION
**Pull request by jackjlli**
Setting force delete to true may cause a race condition where the Controller might write a deleted workflow back to ZK because it's still caching it.

This PR sets this variable to false.

### CODE REVIEW

**Comment by Jackie-Jiang** [comment]
This is changing the behavior of deleting the task queue. Task queue might stuck in bad state and never recover before (might be fixed in new Helix version), so we want to remove it regardless of whether the tasks are finished or not. Let's make sure we can get the expected behavior here.

**Comment by jackjlli** [comment]
Discussed with Helix committer, this is an expected race condition. This happens because the force delete causes inconsistency between the in-memory cache and ZK and shouldn't generally be used. They've agreed to update the JavaDoc.

File: pinot-controller/src/main/java/org/apache/pinot/controller/helix/core/minion/PinotHelixTaskResourceManager.java [hunk 1]
```diff
@@ -139,11 +140,19 @@ public synchronized void resumeTaskQueue(@Nonnull String taskType) {
...
    * @param taskType Task type
    */
-  public synchronized void deleteTaskQueue(@Nonnull String taskType) {
+  public synchronized void deleteTaskQueue(@Nonnull String taskType)
+      throws InterruptedException {
     String helixJobQueueName = getHelixJobQueueName(taskType);
     LOGGER.info("Deleting task queue: {} for task type: {}", helixJobQueueName, taskType);
-    // NOTE: set force delete to true to remove the task queue from ZooKeeper immediately
-    _taskDriver.delete(helixJobQueueName, true);
+    // NOTE: set force delete to true to remove the task queue from ZooKeeper immediately.
```
Comment (sunithabeeram) [inline pinot-controller/src/main/java/org/apache/pinot/controller/helix/core/minion/PinotHelixTaskResourceManager.java]: The comments here don't see to make sense. Can you remove them?

Comment (jackjlli) [reply pinot-controller/src/main/java/org/apache/pinot/controller/helix/core/minion/PinotHelixTaskResourceManager.java]: Done.

**Comment by jackjlli** [comment]
Closing this PR since the flaky API has been commented in integration test.

**Comment by Jackie-Jiang** [comment]
Don't close this one. We want to fix the API and test it in the integration test once Helix side issue is resolved.

**Comment by jackjlli** [comment]
This issue is to track whether the API will be fixed once new Helix release is bumped up in Pinot:
https://github.com/apache/incubator-pinot/issues/4268

After new Helix release gets bumped, we need to uncomment this test:
https://github.com/apache/incubator-pinot/pull/4207

**Pull request closed without merging**